src/formatter/formatComplexDataStructure.js

```diff
@@ -1,14 +1,17 @@
 /* @flow */
 
 import { isValidElement } from 'react';
-import stringify from 'stringify-object';
+import * as stringifyObject from 'stringify-object';
 import sortObject from './sortObject';
 import parseReactElement from './../parser/parseReactElement';
 import formatTreeNode from './formatTreeNode';
 import formatFunction from './formatFunction';
 import spacer from './spacer';
 import type { Options } from './../options';
 
+// Rollup fail to correctly import `stringify-object` so we have to use a workaround
+const stringify = (...args) => stringifyObject.default(...args);
```

Comment:
The `import * as stringifyObject from 'stringify-object';` and this workaround allow us to avoid the rollup issue where @frytyler get stuck into in PR https://github.com/algolia/react-element-to-jsx-string/pull/313